lib/cs_guide/accounts/user.ex

```diff
@@ -26,7 +30,10 @@ defmodule CsGuide.Accounts.User do
   @doc false
   def changeset(user, attrs \\ %{}) do
     user
-    |> cast(attrs, [:email, :password, :admin])
+    |> cast(attrs, [
+      :email, :password, :admin, :role, :verified, :password_reset_token,
+      :password_reset_token_sent_at
```

Comment:
if typing all these out becomes laborious, I like to make use of [word list sigils](https://elixir-lang.org/getting-started/sigils.html#word-lists):

```
|> cast(attrs, ~w(email password admin role verified password_reset_token password_reset_token_sent_at)a)
```